src/whidbey/plugin.clj

```diff
@@ -1,29 +1,98 @@
 (ns whidbey.plugin
+  "This namespace runs inside of Leiningen and rewrites the project map to
+  include the customizations provided by Whidbey."
   (:require
     [leiningen.core.project :as project]))
 
 
-(defn whidbey-profile
-  [options]
-  `{:dependencies [[mvxcvi/puget "0.7.1"]
-                   [mvxcvi/whidbey "0.5.1"]]
+(defn- find-plugin-version
```

Comment:
Yeah, I got tired of keeping the versions in sync. Whidbey's dependency on Puget in the project file will take care of the Puget version as well.